src/pages/chat/component/ChatInput.tsx

```
@@ -31,44 +29,21 @@ const ChatInput = ({
+        <BottomSheet onClose={() => setIsOpen(false)}>
+          <div className="flex flex-col items-start gap-4 px-2 ">
+            {['question1', 'question2', 'question3'].map((key) => (
               <button
                 className="text-sm"
                 onClick={() =>
-                  handleBottomSheetTextClick(
-                    t('chatInput.bottomSheet.question2')
-                  )
+                  handleBottomSheetTextClick(t(`chatInput.bottomSheet.${key}`))
                 }
               >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)